actions/copy/worker.js

```diff
@@ -84,6 +89,7 @@ async function floodgateContent(spToken, adminPageUri, projectExcelPath, project
             status.url = urlInfo.doc.url;
         } catch (error) {
             logger.error(`Error occurred when trying to copy files to floodgated content folder ${error.message}`);
+            throw new Error(`Error occurred when trying to copy files to floodgated content folder : ${error}`);
```

Comment:
In case of an error, I think [throw new Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw) stop further execution without returning the `status`?

We don't want to stop execution if one document failed to copy for some reason.

Reply:
Got you. we don't need to throw error at this location. Copy status check from this promise already has that logic to update status. Removing this and is unncessary.